Nodejs/Product/Nodejs/NpmUI/NpmPackageInstallWindow.xaml

```diff
@@ -44,10 +44,11 @@
                         <Setter Property="AutomationProperties.Name" Value="{Binding Path=Name}" />
                         <Setter Property="AutomationProperties.HelpText">
                             <Setter.Value>
-                                <MultiBinding StringFormat="{} {0}, {1}, {2}">
+                                <MultiBinding StringFormat="{} {0}, {1}, {2}, {3}">
```

Comment:
Feels a bit weird to add this as a parameter binding instead of just the text but I didn't find another way to make it work.